static/adopters.csv

```diff
@@ -109,8 +109,9 @@ Golden Cobra, https://github.com/ikuseiGmbH/Goldencobra
 Google, https://opensource.google/docs/releasing/template/CODE_OF_CONDUCT/
 GorangoCSS, https://gorangocss.kawanua.id/codeofconduct/
 GoReleaser, https://github.com/goreleaser/goreleaser
-Gremlins tracker for Visual Studio Code, https://github.com/nhoizey/vscode-gremlins
```

Comment:
[](https://github.com/ContributorCovenant/contributor_covenant/blob/51eaab51e96f607d688d79066b811f27980d8498/content/version/1/4/code-of-conduct.fa-ir.md)

Reply:
> [](https://github.com/ContributorCovenant/contributor_covenant/blob/51eaab51e96f607d688d79066b811f27980d8498/content/version/1/4/code-of-conduct.fa-ir.md)